code/game/gamemodes/cult/items/tome.dm

```diff
@@ -8,6 +8,70 @@
 	w_class = 2.0
 	unique = TRUE
 	slot_flags = SLOT_BELT
+	var/obj/item/inserted_item  //chalk code graciously stolen from the PDA's pen code   -MalMalmulam
+	var/list/contained_item = list(/obj/item/pen/drafting/) 
+	var/bloody = ""
+
+/obj/item/pen/drafting/red/cult
+	name = "engraved chalk"
+	desc = "A piece of chalk for marking areas of floor, or for drawing.  This one has strange symbols engraved on it."
+	color = COLOR_HUMAN_BLOOD
+	colorName = "redc"	
+
+/obj/item/book/tome/Initialize(mapload, ...)
+	. = ..()
+	inserted_item =	new /obj/item/pen/drafting/red/cult(src)
+	var/list/blooddonors = list()  //the chalk turns the color of a random nearby cultist's blood, if there is one
+	for(var/mob/living/carbon/C in range(1, get_turf(src)))
+		if(!iscultist(C)) 
+			continue
+		blooddonors += C
+		if(blooddonors.len)
+			var/mob/living/carbon/blooddonor = pick(blooddonors)
+			inserted_item.color = blooddonor.species.blood_color
```

Comment:
Instead of doing this, which is pretty hacky, just assign the chalk color when it gets picked up or when it gets used

Reply:
I'll just delete it then.  It defaults to human blood red if no color is assigned, and using it when drawing a rune changes the color to the scribe's blood.